apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -234,7 +236,16 @@ const loadListByChunks = async ({
             throw new Error('Operation timed out')
         }
 
-        newChunk = await list.getAll(context, where, { sortBy, first: chunkSize, skip: skip })
+        // TODO INFRA-538 remove this condition once migration to new server utils to be done
+        const resolvedFields = !isNil(fields) ? fields : 'id'
```

Comment:
Is it possible to make only one place where we say that `'id'` is default fragment?

Reply:
I'll remove this code once migration to be done. This part of code - like a notification for developers: you should rewrite code where you calling loadListByChunks